src/__mocks__/vscode.ts

```diff
@@ -29,6 +27,7 @@ export enum TreeItemCollapsibleState {
     Expanded = 2,
 }
 
+// tslint:disable-next-line: no-namespace
```

Comment:
`// tslint:disable: max-classes-per-file no-namespace` should be reeped and `// tslint:disable-next-line: no-namespace` can be removed